msu/load.nut

```diff
@@ -7,7 +7,10 @@ local function includeLoad( _folder )
 
 includeLoad("ui");
 includeLoad("systems");
+includeLoad("hooks_helper");
```

Comment:
Not sure why we have a folder and then a file that's named the same and include both ish

Reply:
But ok I guess you want to put it into different files later or sth.

Reply:
yeah we might have more stuff I suspect, not a huge fan of how it's organized rn but hey that's for later us to fix when it becomes relevant I think.